scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -300,11 +299,11 @@ sed -i \
     -e 's/"runtimes": {/&\n    "'$__runtime_id'": [\n      "unix", "unix-'$__build_arch'", "any", "base"\n    ],/g' \
 $__frameworkpath/Microsoft.NETCore.App.deps.json
 
-__crossgentimeout=120
+__crossgentimeout=300
 
 function crossgenone(){
     echo $2/crossgen /MissingDependenciesOK /Platform_Assemblies_Paths $2:$3 /in $1 /out $1.ni >$1.log 2>&1
-    timeout $__crossgentimeout $2/crossgen /MissingDependenciesOK /Platform_Assemblies_Paths $2:$3 /in $1 /out $1.ni >>$1.log 2>&1
+    timeout $__crossgentimeout $2/crossgen /MissingDependenciesOK /Platform_Assemblies_Paths $2:$(dirname $1) /in $1 /out $1.ni >>$1.log 2>&1
```

Comment:
Using `/Platform_Assemblies_Paths` set to framework & local directory fixed issues with crossgen failing for Roslyn assemblies.

First crossgen pass is completely clean.